src/page-components/homepage/sections/image-gallery/index.tsx

```diff
@@ -1,131 +1,33 @@
 import React from 'react'
 import * as S from './styles'
-import Item from './item'
-import Img from './img'
+import config from './config.json'
+import Image from './image/index'
+import { POSITIONS } from './helpers'
 
-const imageTreeTest = [
-  {
-    type: 'col',
-    subset: [
-      {
-        type: 'row',
-        subset: [
-          {
-            type: 'img',
-            url: 'https://via.placeholder.com/2000x1200',
-            alt: '...',
-            ratio: 66,
-          },
-          {
-            type: 'img',
-            url: 'https://via.placeholder.com/500',
-            alt: '...',
-            ratio: 33,
-          },
-        ],
-      },
-      {
-        type: 'row',
-        subset: [
-          {
-            type: 'img',
-            url: 'https://via.placeholder.com/15',
-            alt: '...',
-            ratio: 33,
-          },
-          {
-            type: 'img',
-            url: 'https://via.placeholder.com/1110',
-            alt: '...',
-            ratio: 66,
-          },
-        ],
-      },
-    ],
-  },
-  {
-    type: 'col',
-    subset: [
-      {
-        type: 'img',
-        url: 'https://via.placeholder.com/365x500',
-        alt: '...',
-      },
-    ],
-  },
-  {
-    type: 'col',
-    subset: [
-      {
-        type: 'row',
-        subset: [
-          {
-            type: 'img',
-            url: 'https://via.placeholder.com/1500',
-            alt: '...',
-            ratio: 33,
-          },
-          {
-            type: 'img',
-            url: 'https://via.placeholder.com/1500',
-            alt: '...',
-            ratio: 66,
-          },
-        ],
-      },
-      {
-        type: 'row',
-        subset: [
-          {
-            type: 'img',
-            url: 'https://via.placeholder.com/1500',
-            alt: '...',
-            ratio: 2,
-          },
-          {
-            type: 'img',
-            url: 'https://via.placeholder.com/1500',
-            alt: '...',
-            ratio: 1,
-          },
-        ],
-      },
-    ],
-  },
-]
+interface ImageProps {
```

Comment:
udelal bych z toho 2 interfaces, ktere se daj exportovat a prepouzil bych to v `ImageItem` komponentne 

```


export interface ImageProps {
  type: string
  url: string
  alt: string
  position: number
}

export interface ImageGalleryPositions {
  id: number
  colStart: number
  colEnd: number
  rowStart: number
  rowEnd: number
}
```